src/K3poControl.js

```diff
@@ -0,0 +1,354 @@
+'use strict';
+
+var ControlTransportFactory = require(ControlTransportFactory);
```

Comment:
I got an error when running the grunt command 
```
>> Mocha exploded!
>> AssertionError: path must be a string
```
and changing this line with `require('./ControlTransportFactory.js');` resolved it